src/jmcomic/jm_client_interface.py

```
@@ -101,6 +101,14 @@ def __init__(self, resp, ts: str):
         super().__init__(resp)
         self.ts = ts
 
+    # 重写json()方法，专门处理API响应的清理
```

Ruff (0.12.2)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)